src/reference.ts

```diff
@@ -1,5 +1,7 @@
 /// <reference path="../bower_components/svg-typewriter/svgtypewriter.d.ts" />
 
+/// <reference path="core/init.ts" />
```

Comment:
I thought reference.ts defines the order in which the typescript files are compiled / built.  Is this incorrect?